AWSBucketDump.py

```diff
@@ -114,6 +120,20 @@ def downloadFiles(maxsize):
                                 if local_filename =="":
                                         print("Directory..\n")
                                 else:
+                                        if mkdir:
+                                                local_savedir = (line.split('/')[-2]).rstrip()
+                                                # assign new path + filename for result for use in save below
+                                                local_filename = "".join("./%s/%s") % (local_savedir, local_filename)
+                                                # check if dir for host exists, if not create it
+                                                if not os.path.exists("./%s" % local_savedir):
+                                                        print("Creating directory for host: (%s)" % local_savedir)
+                                                        os.mkdir(local_savedir)
+                                                else:
+                                                        print("Using existing directory for host: (%s)" % local_savedir)
+
+                                                print("Saving file to: %s") % local_filename
```

Comment:
`print("Saving file to: %s") % local_filename` should be `print("Saving file to: %s" % local_filename)`